### ISSUE TITLE
Implement EEEJ Analysis

### ISSUE THREAD

**Issue by kflemin** [SEED-platform/seed#4207]

**Comment by RDmitchell** [comment]
@kflemin -- Github automatically closed this issue, and you do have a PR that seems to address it, so it it ok to keep it closed?

**Comment by RDmitchell** [comment]
Instance: staging
SHA: 0dbce5ad6 (2.20.1)
Org: BPS Test DC

@kflemin -- I have looked at the PR for this issue, since there isn't any information in the issue about how this works. 

I have a few questions. 

I used the org that has real DC data, so the addresses are good, and the mapping of the buildings works. 

I turned off everything except Property Points, Census Tracts and Highlight DACs.

If the Census Tract is shaded in blue, presumably that means it is a DAC? 

It might be worth adding a legend or some sort of information to the user so they know that? Maybe even just a Tooltip for the "Highlight DACs" checkbox text? And possibly also for the "Census Tracts" text, to say that the census tracts are outlined (I know it's obvious once you check the box, but sometimes it's nice to tell the user what is going on rather than forcing them to experiment, although in this case, experimenting is very easy). 
![image](https://github.com/SEED-platform/seed/assets/6314950/aba498d9-34c8-4031-aea6-194d36c5df9b)

**Comment by RDmitchell** [comment]
@kflemin -- also, is there a field added to Property to indicate DAC yes or no? So that I could filter on that field and see all the properties that are in DACs?

**Comment by RDmitchell** [comment]
@isalanglois -- I need to document this in the release notes for 2.20, and so can you give me some background about how the information about DACs might be used? Would it be used to identify areas for energy retrofits, etc?

**Comment by RDmitchell** [comment]
@kflemin -- I am trying to document for users how the EEEJ feature works, and I want to make sure I have this information correct. 

It seems like SEED is using the US Census Bureau data to define the census tracts
https://geocoding.geo.census.gov/geocoder/geographies/onelineaddress?form to compare)

And the 
![image](https://github.com/SEED-platform/seed/assets/6314950/660a876c-cdee-40a5-8efe-0f52646335ec)
for the DAC definitions
 https://screeningtool.geoplatform.gov/en/#3/33.47/-97.5 to compare)

Is that correct? Thx !!

**Comment by RDmitchell** [comment]
@kflemin -- another question -- does SEED make a DAC field with a Yes or No in it so that those buildings can be filtered in the Inventory List and Detail? 

If not, we should add that.

**Comment by RDmitchell** [comment]
I guess alternatively a DAC label could be added to all the records, which I am now trying to do from the map, and then going to the detail view for each inventory record. The map is painfully slow to load (DC data, 3,000+ records) and every time it loads with the original settings, instead of those that I set before I went to the building Detail View. 

I guess I will make some new issues for these.
